src/utils/constants/apiEndPoints.jsx

```diff
@@ -1,6 +1,17 @@
-const SIGNUP_API = '/api/auth/signup'
-const LOGIN_API = '/api/auth/login'
-const GET_VIDEOS = '/api/videos'
-const GET_CATEGORIES = '/api/categories'
+const SIGNUP_API = "/api/auth/signup";
+const LOGIN_API = "/api/auth/login";
+const GET_VIDEOS = "/api/videos";
+const GET_CATEGORIES = "/api/categories";
+const POST_HISTORY = "/api/user/history";
+const POST_WATCH_LATER = "/api/user/watchlater";
+const DELETE_WATCH_LATER = "/api/user/watchlater/:videoId";
 
-export { SIGNUP_API, LOGIN_API, GET_VIDEOS, GET_CATEGORIES}
\ No newline at end of file
+export {
+  SIGNUP_API,
+  LOGIN_API,
+  GET_VIDEOS,
+  GET_CATEGORIES,
+  POST_HISTORY,
+  POST_WATCH_LATER,
+  DELETE_WATCH_LATER,
+};

```

Comment:
Appreciation:- I like that you have made the constants for API endpoints in a separate file.

Reply:
Thank you :D